modules/op-running-unpriveleged-builds-in-a-container.adoc

```diff
@@ -0,0 +1,45 @@
+[id='op-running-unpriveleged-builds-in-a-container._{context}']
+= Running unprivileged builds in a container
+
+You can run builds as a non-root user inside a container. To run as a non-root user inside the container, you need your container to do some set up to add the `subuid` and `subgid` with the latest shadow-utils.
+
+The public image based on Red Hat UBI has everything already set up for us in its https://catalog.redhat.com/software/containers/ubi8/buildah/602686f7b16b1eb2e30807ee?container-tabs=dockerfile[Containerfile]. As an extra step, you need to “force” running the container as the “build” user since if we leave it to the default, it will be “randomed” and would not have the setup needed to run as non-root.
+
+You can directly edit or export to a specific namespace the buildah ClusterTask as described in the previous section. You can add this `securityContext`` to each https://github.com/tektoncd/operator/blob/main/cmd/openshift/operator/kodata/tekton-addon/addons/02-clustertasks/buildah/buildah-task.yaml#L63[build] and https://github.com/tektoncd/operator/blob/main/cmd/openshift/operator/kodata/tekton-addon/addons/02-clustertasks/buildah/buildah-task.yaml#L75[push] steps.
+
+[source,yaml,subs="attributes+"]
+----
+securityContext:
+    runAsUser: 1000
+----
+
+Since we are building as user, buildah will keep its image caching in `/home/build/.local/cache`. Hence instead of volume mounting `/var/lib/` containers we are going to change the `mountPath` on build and push steps to the user one :
```

Comment:
```suggestion
Because you are building as a local user, buildah saves its image cache to `/home/build/.local/cache`. Thus, instead of  mounting the containers located in the `/var/lib/` directory, change the `mountPath` for the build to your local user container directory by changing the value for `mountPath` as follows:
```
- The first sentence doesn't make sense to me in this context. Why does the user need to know about where the cache directory is located when you're talking about the local `volumeMount` path?
- Per ISG: Do not use "since" to mean "because".
- Why are these sentences suddenly using "we"? You need to rephrase so that the sentence becomes "you"-centric. I've suggested one possible way of achieving this.
- Per ISG: Avoid future tense